crates/sparrow-session/src/tests.rs

```diff
@@ -1,2 +0,0 @@
-//! Partitioned execution tests
```

Comment:
Nothing was referencing this -- the `mod tests` in partitioned.rs corresponded to `partitioned/tests.rs` which was missing.